packages/studio-plugin/src/createStudioPlugin.ts

```diff
@@ -52,6 +53,11 @@ export default async function createStudioPlugin(
   const localDataMappingManager = studioConfig.isPagesJSRepo
     ? new LocalDataMappingManager(studioConfig.paths.localData)
     : undefined;
+  // TODO: Remove false-gate and pass real API key once we get it
+  const managementApiService =
+    false && studioConfig.isPagesJSRepo
+      ? new ManagementApiService("apiKey")
+      : undefined;
```

Comment:
I think we can remove this block of code for now, it shouldn't be hard for somebody to come back to this PR and get it again
```  
const managementApiService =
    false && studioConfig.isPagesJSRepo
      ? new ManagementApiService("apiKey")
      : undefined;
```

Reply:
Sure, I removed it from `createStudioPlugin` for now, but I left in the `configureStudioServer` logic